tests/EdFi.Ods.Api.IntegrationTests/AuthorizationViewHelper.cs

```diff
@@ -7,36 +7,69 @@
 using System;
 using System.Collections.Generic;
 using System.Data;
+using System.Linq;
+
+
 namespace EdFi.Ods.Api.IntegrationTests
 {
     public static class AuthorizationViewHelper
     {
 
-        private static bool QueryAuthorizationByViewName(IDbConnection connection, string  viewName)
+        public static void ShouldNotContainDuplicate(IDbConnection connection,
```

Comment:
I see a little later on how you're passing in the expected count to try to avoid "duplicates", but that doesn't check for duplicates, that just checks that you got back as many records as you expected.  Your logic was fine before doing the group by and checking to make sure there weren't any groups with more than 1, you could even have the helper method just take in a list of tuples and apply the grouping and Should checks.

Reply:
I updated it as discussed